cli/azd/pkg/apphost/generate.go

```diff
@@ -383,7 +383,7 @@ func GenerateProjectArtifacts(
 	projectFileContext := genProjectFileContext{
 		Name: projectName,
 		Services: map[string]string{
-			"app": fmt.Sprintf(".%s%s", string(filepath.Separator), appHostRel),
+			"app": fmt.Sprintf("./%s", filepath.ToSlash(appHostRel)),
```

Comment:
Could you include tests that include both azure.yaml paths in both ways to ensure we don't break compat?

Reply:
Wrote some directed tests.